app/controllers/membership_applications_controller.rb

```diff
@@ -6,7 +6,10 @@ def new
   end
 
   def create
-    @membership_application = current_user.membership_applications.new(membership_application_params)
+    # must create the membership_application before using the policy to check the parameters. @see https://www.sitepoint.com/straightforward-rails-authorization-with-pundit/
+    @membership_application = current_user.membership_applications.new
```

Comment:
Unless this code is WIP I would like to ask you to remove comments from the code.